sphinx/ext/viewcode.py

```diff
@@ -68,10 +68,7 @@ def is_supported_builder(builder: Builder) -> bool:
         return False
     elif builder.name == 'singlehtml':
         return False
-    elif builder.name.startswith('epub') and not builder.config.viewcode_enable_epub:
-        return False
-    else:
-        return True
+    return not (builder.name.startswith('epub') and not builder.config.viewcode_enable_epub)
```

Comment:
Indeed, this is equivalent to the current code. But this changes the meaning of the code, I feel. I think it's not good to expand consecutive else-if block to such code.

Reply:
done